### PR TITLE
Add 4.2.0.

### PR DESCRIPTION
**Pull request by sf-tyler-jeong**

### CODE REVIEW

**Review by OnestarLee — APPROVED** [review]

File: pubspec.yaml [hunk 1]
```diff
@@ -30,6 +32,10 @@ dependencies:
   web_socket_channel: ^2.4.0
   universal_io: ^2.2.0
   async: ^2.10.0
+  isar: *isar_version
+  isar_flutter_libs: *isar_version
```
Comment (asaarnak) [inline pubspec.yaml:36]: After updating to 4.2.0 we get this issue:
https://github.com/isar/isar/issues/1470

```

1: Task failed with an exception.
-----------
* What went wrong:
A problem occurred configuring project ':isar_flutter_libs'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Specify a namespace in the module's build file. See https://d.android.com/r/tools/upgrade-assistant/set-namespace for information about setting the namespace.

     If you've specified the package attribute in the source AndroidManifest.xml, you can use the AGP Upgrade Assistant to migrate to the namespace value in the build file. Refer to https://d.android.com/r/tools/upgrade-assistant/agp-upgrade-assistant for general information about using the AGP Upgrade Assistant.```

Comment (asaarnak) [reply pubspec.yaml:36]: @OnestarLee @sf-tyler-jeong
isar version 3+ dependency does not work with Android Gradle Plugin 8.0+
We pinned our sendbird_chat_sdk to 4.1.0 version to be able to build project.

Comment (sf-tyler-jeong) [reply pubspec.yaml:36]: @asaarnak 
Thank you for the information. If there is any update for isar regarding this, we will apply it.
Alternativiely, if you downgrade the gradle version to 7.x, is it okay?

Comment (asaarnak) [reply pubspec.yaml:36]: @sf-tyler-jeong 
Should be ok on 7.x as this is 8+ only related breaking change.
We downgraded sendbird_chat_sdk instead as we have been using AGP 8+ over half a year.

Comment (asaarnak) [reply pubspec.yaml:36]: isar v4 supports AGP 8+ and also web, but it only has dev releases.

Comment (asaarnak) [reply pubspec.yaml:36]: @sf-tyler-jeong @OnestarLee 
`isar-community` has been created to keep isar v3 up to date.
https://github.com/isar-community/isar

Comment (asaarnak) [reply pubspec.yaml:36]: Discussions about `isar` future: https://github.com/isar/isar/discussions/1498 https://github.com/isar/isar/discussions/1581

Comment (sf-tyler-jeong) [reply pubspec.yaml:36]: @asaarnak
I think we need to change it to `isar-community` at an appropriate time.
Thanks for the information!

Comment (asaarnak) [reply pubspec.yaml:36]: @sf-tyler-jeong Any news when you plan to switch to maintained isar dependency?

Comment (asaarnak) [reply pubspec.yaml:36]: @sf-tyler-jeong @OnestarLee  Any news about fixing the `isar version 3+ dependency does not work with Android Gradle Plugin 8.0+` issue?

Comment (mithatns) [reply pubspec.yaml:36]: Any news about this? i need use for Android Gradle Plugin 8.0+

**Pull request merged**